libs/ngrx-toolkit/src/lib/devtools/internal/devtools-syncer.service.ts

```diff
@@ -8,7 +8,7 @@ import { DevtoolsInnerOptions } from './devtools-feature';
 import { Connection, StoreRegistry, Tracker } from './models';
 
 const dummyConnection: Connection = {
-  send: () => void true,
+  send: () => true,
 };
 
```

Comment:
Added a separate test file.  I did look at other tests and noticed deprecated use of `TestBed.flushEffects()` should be able to quickly replaced with `TestBed.tick()`.  Best to fix during Vitest conversion PR.